template/src/config/store.ts

```diff
@@ -13,7 +13,9 @@ export default function (initialState = {}): Store {
   const store = configureStore({
     reducer: createReducer(),
     preloadedState: initialState,
-    middleware: getDefaultMiddleware().concat(middlewares),
+    middleware: getDefaultMiddleware({
+      serializableCheck: false
```

Comment:
This way you ignore all checks - I don't think we want this.
The only one we need to ignore is the one coming from `promises` inside actions - you can use something like this to only ignore this specific one:

```
middleware: getDefaultMiddleware({
    serializableCheck: {
      ignoredActionPaths: ['meta.promise'],
    }
})
```